tests/check_check_sub.c

```diff
@@ -37,6 +37,40 @@ double NAN;
 #if !defined(INFINITY)
 double INFINITY;
 #endif
+#if !defined(isnan)
```

Comment:
Can this be put into the lib dir as a reimplementation of these if they do not exist?

Reply:
See timer_settime as an example.

Reply:
I will try (https://github.com/libcheck/check/pull/129/commits/37a1188e9c23fbebec4829c3f105fdf2977f4277)